.test.sh

```diff
@@ -21,7 +21,9 @@ find src/ -name '*.pyc' -delete
 rm -f build/junit.xml
 
 # testing management commands that require a queue shared between processes
-LAX_MULTIPROCESSING=1 coverage run --source='src/' --omit='*/tests/*,*/migrations/*' src/manage.py test "$module" --no-input
+export DJANGO_SETTINGS_MODULE=core.settings
+export LAX_MULTIPROCESSING=1
+pytest "$module" -vvv --cov=src --cov-config=.coveragerc
```

Comment:
pytest pathing differs from the default python and green test runners but it's still the first arg